detections/application/cisco_duo_admin_login_unusual_browser.yml

```diff
@@ -0,0 +1,68 @@
+description: The following analytic identifies instances where a Duo admin logs in using a browser other than Chrome, which is considered unusual 
+  based on typical access patterns. Please adjust as needed to your environment. The detection leverages Duo activity logs ingested via the Cisco 
+  Security Cloud App and filters for admin login actions where the browser is not Chrome. By renaming and aggregating relevant fields such as user, 
+  browser, IP address, and location, the analytic highlights potentially suspicious access attempts that deviate from the norm. This behavior is 
+  significant for a SOC because the use of an unexpected browser may indicate credential compromise, session hijacking, or the use of unauthorized 
+  devices by attackers attempting to evade detection. Detecting such anomalies enables early investigation and response, helping to prevent privilege 
+  escalation, policy manipulation, or further compromise of sensitive administrative accounts. The impact of this attack could include unauthorized 
+  changes to security policies, user access, or the disabling of critical security controls, posing a substantial risk to the organizations security posture.
+search: '`cisco_duo_activity` "action.name"=admin_login NOT access_device.browser IN (Chrome)
+  | rename actor.name as user access_device.ip.address as src_ip
+  | stats count min(_time) as firstTime max(_time) as lastTime by access_device.browser 
+  access_device.browser_version src_ip access_device.location.city 
+  access_device.location.country access_device.location.state access_device.os access_device.os_version 
+  actor.details actor.type outcome.result user
+  | `security_content_ctime(firstTime)` | `security_content_ctime(lastTime)`
+  | `cisco_duo_admin_login_unusual_browser_filter`'
+how_to_implement: The analytic leverages Duo activity logs to be ingested using the
+  Cisco Security Cloud App (https://splunkbase.splunk.com/app/7404).
+known_false_positives: unknown
+references:
+- https://splunkbase.splunk.com/app/7404
+drilldown_searches:
+- name: View the detection results for - "$user$"
+  search: '%original_detection_search% | search  user = "$user$"'
+  earliest_offset: $info_min_time$
+  latest_offset: $info_max_time$
+- name: View risk events for the last 7 days for - "$user$"
+  search: '| from datamodel Risk.All_Risk | search normalized_risk_object IN ("$user$")
+    starthoursago=168  | stats count min(_time) as firstTime max(_time) as lastTime
+    values(search_name) as "Search Name" values(risk_message) as "Risk Message" values(analyticstories)
+    as "Analytic Stories" values(annotations._all) as "Annotations" values(annotations.mitre_attack.mitre_tactic)
+    as "ATT&CK Tactics" by normalized_risk_object | `security_content_ctime(firstTime)`
+    | `security_content_ctime(lastTime)`'
+  earliest_offset: $info_min_time$
+  latest_offset: $info_max_time$
+rba:
+  message: A user $user$ has logged in using an unusual browser"
```

Comment:
extra " in message for all detections. Can you also enhance this message to include  `$access_device.browser$` for this and all other detections?

Reply:
Did it for Cisco Duo Activity logs ones which contains the browser field